src/test/java/study/StringTest.java

```diff
@@ -0,0 +1,53 @@
+package study;
+
+import org.assertj.core.api.Assertions;
+import org.junit.jupiter.api.DisplayName;
+import org.junit.jupiter.api.Test;
+
+public class StringTest {
+    @Test
+    @DisplayName("'1,2'가 입력되었을 때, 1과 2로 분리된다.")
+    public void splitTwoInputNumberTest() {
+        String[] result = "1,2".split(",");
+
+        Assertions.assertThat(result).containsExactly("1", "2");
+    }
+
+    @Test
+    @DisplayName("'1'이 입력되었을 때, 1만을 포함하는 배열을 반환한다.")
+    public void splitOneInputNumberTest() {
+        String[] result = "1".split(",");
+
+        Assertions.assertThat(result).containsExactly("1");
+    }
+
+    @Test
+    @DisplayName("'(1,2)' 값이 주어졌을 때 substring() 메소드를 사용해 ()을 제거한다.")
+    public void subStringTest() {
+        String result = "(1,2)".substring(1,4);
+
+        Assertions.assertThat(result).isEqualTo("1,2");
+    }
+
+    @Test
+    @DisplayName("'abc' 값이 주어졌을 때, String의 charAt() 메소드를 활용해 특정 위치의 문자를 가져온다")
+    public void getSpecificCharacterByCharAtTest() {
+        char a = "abc".charAt(0);
+        char b = "abc".charAt(1);
+        char c = "abc".charAt(2);
+
+        Assertions.assertThat(a).isEqualTo('a');
+        Assertions.assertThat(b).isEqualTo('b');
+        Assertions.assertThat(c).isEqualTo('c');
```

Comment:
```suggestion
        Assertions.assertThat("abc".charAt(0)).isEqualTo('a');
        Assertions.assertThat("abc".charAt(1)).isEqualTo('b');
        Assertions.assertThat("abc".charAt(2)).isEqualTo('c');
```
이렇게 단순한 test case의 경우 로컬 변수 없이 위와 같이 구현해 보는 것은 어떨까?